src/mender-configure

```diff
@@ -37,7 +38,9 @@ safe_copy() {
         echo "safe_copy can only handle one file copy at a time" 1>&2
         exit 2
     fi
-    cp "$1" "$2".tmp
+    touch "$2".tmp
+    chmod $DEFAULT_PERMISSIONS "$2".tmp
+    cat "$1" > "$2".tmp
```

Comment:
What is the difference from this to:
```
cp "$1" "$2".tmp
chmod $DEFAULT_PERMISSIONS "$2".tmp
```
?

Reply:
I didn't want to create the file in the target folder with the content we want to protect, readable by everybody, not even for a single moment. That's why I'm creating the empty file, changing the permissions, then copying the content over. Does it make sense?

Reply:
It makes sense, thanks for the clarification.